atomic_reactor/inner.py

```diff
@@ -588,7 +588,7 @@ class ImageBuildWorkflowData(ISerializer):
 
     # List of RPMs that go into the final result, as per utils.rpm.parse_rpm_output
     # Each RPM inside is a mapping containing the name, version, release and other attributes.
-    image_components: List[Dict[str, Union[int, str]]] = None
+    image_components: Optional[List[Dict[str, Union[int, str]]]] = None
```

Comment:
because there are code pars that checks `is None`

```
atomic_reactor/plugins/post_rpmqa.py:        if self.workflow.data.image_components is not None:
tests/plugins/test_rpmqa.py:    assert workflow.data.image_components is None
tests/plugins/test_rpmqa.py:        assert workflow.data.image_components is None
tests/plugins/test_rpmqa.py:        assert workflow.data.image_components is None
tests/plugins/test_rpmqa.py:            assert workflow.data.image_components is None
tests/plugins/test_rpmqa.py:            assert workflow.data.image_components is None
tests/plugins/test_rpmqa.py:        assert workflow.data.image_components is None

```

I didn't want to break any well hidden logic